### DISCUSSION TITLE
Regarding Whisper AI model transcribing the text into Spanish even if the audio is in English language.

### DISCUSSION THREAD

**Discussion by yashthigale** [openai/whisper#1736] [Q&A]
I am working on the Whisper AI model where an audio file is transcribed into text and the default language would be English. But sometimes the language in the audio file is English and the transcribed text happens to be in Spanish. Even if you run the file multiple times, the result is the same. Can someone help me on this

**Comment by coastal45** [comment]
Are you explicitly stating English in your command?
I have noticed isolated cases where Whisper seems to pop into translation mode momentarily. When I've seen this, it is a very common phrase and there is a correspondence in meaning between the audio and the returned wrong language word.
Possibly in your case?

**Reply by yashthigale** [reply]
Yeah, I have explicitly stated English as my language in my code. But still I am facing the same issue

**Reply by coastal45** [reply]
Is there a correspondence in meaning between the audio and the returned wrong language word? And if so, is it a common word/phrase?

**Reply by yashthigale** [reply]
Hello. I changed the whole code and this particular issue has been resolved. Thank you so much for your support.

**Reply by noaonoszko** [reply]
I'm having the same issue, what was the solution? For me, it translates to Spanish from a variety of languages although I specify the correct language in the API call.
